### PR TITLE
sway: update to 1.8.1.

### PR DESCRIPTION
**Pull request by mhmdanas**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/sway/template [hunk 1]
```diff
@@ -14,7 +14,7 @@ maintainer="Olaf Mersmann <olafm@p-value.net>"
 license="MIT"
 homepage="https://swaywm.org"
```
Comment (paper42) [inline srcpkgs/sway/template:15]:
```suggestion
homepage="https://swaywm.org"
changelog="https://github.com/swaywm/sway/releases"
```

File: srcpkgs/sway/template [hunk 2]
```diff
@@ -13,8 +13,9 @@ short_desc="Tiling Wayland compositor compatible with i3"
 maintainer="Olaf Mersmann <olafm@p-value.net>"
```
Comment (paper42) [inline srcpkgs/sway/template:13]: this person last contributed in 2021, could you orphan this package? (feel free to adopt it if you are actively using it)

Comment (mhmdanas) [reply srcpkgs/sway/template]: I use Sway actively, so I'm fine with adopting it. @olafmersmann is this fine with you?

Comment (olafmersmann) [reply srcpkgs/sway/template]: Sure!

**Pull request merged**